app/views/brexit_checker/_action_list.html.erb

```diff
@@ -15,6 +15,13 @@
           <%= action.title %>
         <% end %>
       </p>
+      <% if action.lead_time.present? %>
+        <div class="govuk-body govuk-!-font-weight-bold">
```

Comment:
These override classes should only be used sparingly as they add `!important` to styles. We should move this styling to a new class.

Reply:
That's gone!